tests/test_tool_runtime_records.py

```
@@ -0,0 +1,207 @@
+def test_tool_result_record_keeps_metadata_and_bounds_output_preview() -> None:
+    full_output = "x" * 900
+    result = ToolResult(
+        call_id="call_cmd",
+        name="execute_command",
+        status="completed",
+        output=full_output,
+        started_at=10.0,
+        ended_at=12.0,
+        truncated=True,
+        truncation_direction="tail",
+        artifact_path="/tmp/tool-output-call_cmd.txt",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/tool-output-call_cmd.txt"

(S108)